docs/en/latest/how-to-build.md

```diff
@@ -37,11 +37,13 @@ This installation method is suitable for CentOS 7. For now, the Apache APISIX RP
 
 ```shell
 sudo yum-config-manager --add-repo https://repos.apiseven.com/packages/centos/apache-apisix.repo
-# View apisix package information, only 2.10.0 is included for now
+# View the information of the latest apisix 2.10.1-0.el7 package
```

Comment:
We can hide the version of apisix? Otherwise, we need to update it each time after the release.

Reply:
> We can hide the version of apisix? Otherwise, we need to update it each time after the release.

@spacewander good catch. Done. Thanks.